guardrails/guard.py

```diff
@@ -371,7 +374,7 @@ def _call_sync(
         call_log: Call,
         *args,
         **kwargs,
-    ) -> ValidationOutcome[OT]:
+    ) -> Union[ValidationOutcome[OT], Iterable]:
```

Comment:
Can Iterable be typed? i.e. is `Iterable[ValidationOutcome[OT]]` valid?

Reply:
Yes. Changed it to `Iterable[str]`, as we return a generator of strings where each string is a concatenation of the raw and validated response.